packages/fxa-auth-server/package.json

```diff
@@ -141,6 +148,7 @@
     "load-grunt-tasks": "^5.1.0",
     "lodash.chunk": "4.2.0",
     "lodash.pick": "4.4.0",
+    "maildev": "^1.1.0",
```

Comment:
Can I get you to commit the changes to the `yarn.lock` file after a `yarn install`?

Reply:
Yeah I'll definitely add it with my next commit